tools/landmark_registration/landmark_registration.xml

```diff
@@ -1,41 +1,50 @@
-<tool id="ip_landmark_registration" name="Landmark Registration" version="0.0.2">
-    <description>Landmark Registration</description>
+<tool id="ip_landmark_registration" name="Landmark Registration" version="0.0.3" profile="20.05">
+    <description>estimates the affine transformation matrix</description>
     <requirements>
-        <requirement type="package" version="0.14.2">scikit-image</requirement>
-        <requirement type="package" version="0.23.4">pandas</requirement>
-        <requirement type="package" version="1.15.4">numpy</requirement>
-        <requirement type="package" version="0.15.1">tifffile</requirement>
+        <requirement type="package" version="0.18.1">scikit-image</requirement>
+        <requirement type="package" version="1.6.2">scipy</requirement>
+        <requirement type="package" version="1.2.4">pandas</requirement>
+        <requirement type="package" version="1.20.2">numpy</requirement>
     </requirements>
-    <command><![CDATA[ 
-         python '$__tool_directory__/landmark_registration.py'
-         --residual_threshold $residual_threshold
-         --max_trials $max_trials
-         '$points_file1'
-         '$points_file2'
-         '$warp_matrix'
+    <command detect_errors="aggressive">
+    <![CDATA[ 
+        python '$__tool_directory__/landmark_registration.py'
+        '$fn_pts1'
+        '$fn_pts2'
+        '$fn_tmat'
```

Comment:
Mh, I do not understand res_thr and max_iter are missing now. The version yesterday looked better, anything I can help?

Reply:
Yesterday's version did not pass the checks. Now I am trying to locate the bugs. 

"planemo test --biocontainers <tool>" sometimes does not work on my local machine. Then it is hard for me to find out where the problems are.

Reply:
why does planemo test does not work for you? Can you give me an error report?

Reply:
<img width="1549" alt="Screen Shot 2022-02-20 at 14 59 05" src="https://user-images.githubusercontent.com/64555893/154846279-513ad22c-8d1e-4c71-a235-9fd8cf97751e.png">

Here is an example of the error report after running "planemo test --biocontainers" on my local machine. And this happened after all checks have passed on GitHut.

Reply:
Sometimes the errors were different. For example, before updating the versions of packages, the error was "No module named 'pandas'".

Reply:
This is not a problem of --biocontainers. It just tells you that the container could not be built successfully. This indicates that the environment could not be resolved and if it's working for you locally, you are probably using local libraries. Using --biocontainers is probably a good way to avoid that.